src/TodoListItem.js

```diff
@@ -1,8 +1,13 @@
 import React from "react";
 
-function TodoListItem({ todo }) {
+function TodoListItem({ todo, onRemoveTodo }) {
+
+    function onClick(){
+        onRemoveTodo(todo.id);
+    }
+
     return (
-        <li>{todo.title}</li>
+        <li>{todo.title} <button onClick={onClick}>Remove</button></li>
```

Comment:
Nice once-liner with your handler! But you can already see how it might be a little confusing with the two `onClick` functions!